protocol/contracts/leaser/src/msg.rs

```diff
@@ -79,6 +84,20 @@ pub enum SudoMsg {
         lease_position_spec: PositionSpecDTO,
         lease_due_period: Duration,
     },
+    CloseProtocol {
+        migration_spec: <HigherOrderGranularity<HigherOrderProtocolContracts, HigherOrderOption> as HigherOrderType>::Of<MigrationSpec>,
```

Comment:
I'd say that it should rather be: `ProtocolContracts<MigrationSpec>`, as all contracts should be migrated into their respective blank counterparts, rather than just some.

Reply:
thanks, nice simplification .. and a better API